examples/gallery/maps/roads.py

```diff
@@ -0,0 +1,42 @@
+"""
+Roads 
+--------------------
+The :meth:`pygmt.Figure.plot` method allows to plot geographical data which is
+stored in a geopandas.GeoDataFrame object.
+"""
+
+import geopandas as gpd
+import pygmt
+
+# read shapefile data using geopandas
+gdf = gpd.read_file(
+    "http://www2.census.gov/geo/tiger/TIGER2015/PRISECROADS/tl_2015_15_prisecroads.zip"
+)
+
+# the dataset contains different road types listed in the RTTYP column,
+# here we select the following ones to plot:
+# Common name roads
+roads_common = gdf[gdf.RTTYP == "M"]
+# State recognized roads
+roads_state = gdf[gdf.RTTYP == "S"]
+# Interstates
+roads_interstate = gdf[gdf.RTTYP == "I"]
+
+fig = pygmt.Figure()
+
+# Define target region around O'ahu (Hawai'i)
+region = [-158.3, -157.6, 21.2, 21.75]
+
+fig.basemap(region=region, projection="M12c", frame=True)
```

Comment:
Add a title perhaps. I couldn't find a way to escape the `'` to make it Hawai'i (because of #247), but maybe you can find a way.

```suggestion
fig.basemap(
    region=region, projection="M12c", frame=["af", 'WSne+t"Main roads of Hawaii"']
)
```